bundle/config/root_test.go

```diff
@@ -163,3 +167,60 @@ func TestRootMergeEnvironmentWithMode(t *testing.T) {
 	require.NoError(t, root.MergeEnvironment(env))
 	assert.Equal(t, Development, root.Bundle.Mode)
 }
+
+func TestConfigFileNames_FindInPath(t *testing.T) {
+	testCases := []struct {
+		name     string
+		files    []string
+		expected string
+		err      string
+	}{
+		{
+			name:     "file found",
+			files:    []string{"databricks.yaml"},
+			expected: "BASE/databricks.yaml",
+			err:      "",
+		},
+		{
+			name:     "multiple files found",
+			files:    []string{"databricks.yaml", "bundle.yaml"},
+			expected: "",
+			err:      "multiple bundle root configuration files found",
+		},
+		{
+			name:     "file not found",
+			files:    []string{},
+			expected: "",
+			err:      "no such file or directory",
+		},
+	}
+
+	if runtime.GOOS == "windows" {
+		testCases[2].err = "The system cannot find the file specified."
+	}
+
+	cfn := ConfigFileNames{"databricks.yaml", "databricks.yml", "bundle.yaml", "bundle.yml"}
```

Comment:
This should be equal to `FileNames` defined in the main source file (can refer to it).

Otherwise, that could change and this test wouldn't catch it.